sources/packages/backend/apps/api/src/services/application-bulk-withdrawal/application-bulk-withdrawal-import-business-validation.models.ts

```diff
@@ -0,0 +1,310 @@
+import {
+  Equals,
+  IsBoolean,
+  IsDate,
+  IsEnum,
+  IsIn,
+  IsNumberString,
+  Length,
+  ValidateIf,
+  ValidationArguments,
+} from "class-validator";
+import {
+  APPLICATION_NUMBER_LENGTH,
+  ApplicationStatus,
+  SIN_NUMBER_LENGTH,
+} from "@sims/sims-db";
+import {
+  ApplicationBulkWithdrawalHeader,
+  ApplicationWithdrawalImportTextModel,
+  DataTextHeaders,
+  RecordType,
+} from "./application-bulk-withdrawal-import-text.models";
+
+const userFriendlyNames = {
```

Comment:
The idea of having a similar approach for the offerings was to make field names, that are internally named differently, to be more "user friendly". It seems that the idea here is not the same.
Also, please review if all the items are used, for instance, I believe that `isCompleted` is not used.